docs/mp/faulttolerance/01_overview.adoc

```diff
@@ -30,30 +32,27 @@ service restarts, network delays, temporal infrastructure instabilities, etc.
 
 == Fault Tolerance in Helidon
 
-Fault Tolerance method annotations are internally implemented using _commands_.
-Essentially, each Fault Tolerance method invocation runs under the supervision
-of one of these commands. Command execution can be affected by certain properties:
-some defined by the Fault Tolerance specification and others internal to Helidon.
+The latest implementation of MP Fault Tolerance is built on top of Helidon's SE
+Fault Tolerance. Thus, some configuration for Helidon SE Fault
+Tolerance also applies to MP. The next section describes some
+configuration properties that are of particular interest to MP applications.
 
-== Fault Tolerance Configuration
-This section describes a few configuration parameters that are specific to Helidon's Fault
-Tolerance implementation. See the {fault-tolerance-spec-url}[Fault Tolerance MicroProfile API] for more information.
+=== Configuration
 
-The following is a list of config properties supported by Helidon that are
-_not_ part of the Fault Tolerance specification:
+Helidon's implementation uses two types of thread pools: normal and scheduled. The default
+core size of these executors is 16; however, that can be configured using an `application.yaml`
+file as follows:
 
-- `fault-tolerance.commandThreadPoolSize`: Asynchronous commands require the use of a
-separate thread pool. Threads created in this pool are named `helidon-ft-async-<N>`.
-This config property controls the size of such a pool. The default size is 8.
+[source,yaml]
```

Comment:
Does this work via `microprofile-config.properties` ? should we be showing that instead? If not then maybe we should mention that you must use applicationl.yaml.

Reply:
It does not at the moment, I can clarify that.